ciao-cli/workload.go

```diff
@@ -256,6 +257,52 @@ func optToReq(opt workloadOptions, req *types.Workload) error {
 	return nil
 }
 
+func outputWorkload(w types.Workload) {
+	var opt workloadOptions
+
+	opt.Description = w.Description
+	opt.VMType = string(w.VMType)
+	opt.FWType = w.FWType
+	opt.ImageName = w.ImageName
+	opt.ImageID = w.ImageID
+	for _, d := range w.Defaults {
+		if d.Type == payloads.VCPUs {
+			opt.Defaults.VCPUs = d.Value
+		} else if d.Type == payloads.MemMB {
+			opt.Defaults.MemMB = d.Value
+		}
+	}
+
+	for _, s := range w.Storage {
+		d := disk{
+			Size:      s.Size,
+			Bootable:  s.Bootable,
+			Ephemeral: s.Ephemeral,
+		}
+		if s.ID != "" {
+			d.ID = &s.ID
+		}
+
+		src := source{
+			Type: s.SourceType,
+			ID:   s.SourceID,
+		}
+
+		d.Source = &src
+
+		opt.Disks = append(opt.Disks, d)
+	}
+
+	b, err := yaml.Marshal(opt)
+	if err != nil {
+		fatalf(err.Error())
+	}
+
+	fmt.Println(string(b))
+	fmt.Println(w.Config)
+
```

Comment:
@markdryan thanks for catching that, I pushed an updated commit without the extraneous newline.